crates/editor_ui/Cargo.toml

```diff
@@ -20,18 +20,23 @@ space_prefab.workspace = true
 space_undo.workspace = true
 space_persistence.workspace = true
 space_shared.workspace = true
+serde.workspace = true
 
 bevy_egui = "0.24"
 egui-gizmo = "0.13"
 egui_dock = "0.9"
 
 bevy_debug_grid = "0.4"
 
-bevy_mod_picking = { version = "0.17", default-features = false, features = [
+bevy_mod_picking = { git = "https://github.com/dgriffith0/bevy_mod_picking.git", branch = "pick_hidden", default-features = false, features = [
     "backend_raycast",
     "selection",
 ] }
+bevy_asset_loader = "0.19"
 bevy_panorbit_camera = "0.9"
-bevy_sprite3d = "2.7.0"
+bevy_mod_billboard = "0.5.1"
 pretty-type-name = "1.0.1"
 convert_case = "0.6"
+
+anyhow = "1.0"
+bevy_common_assets = { version = "0.8", features = ["ron"] }
```

Comment:
I think our prefab saving plugin should be able to deal with this info. Did you try it?

Reply:
I did not yet. I haven't done any digging into that side of things fully but I can do so in a few here.

Reply:
Tapping back in, do you mean the functionality to save a scene with the new lights and cameras and reloading that later? If so then yes it does seem to work just fine for me.

Reply:
I just realized I completely misunderstood what the question was. I have not yet tried using the prefab plugin for scenes, but I can look into that shortly. What I had tested was saving a scene with lights and cameras in it and reloading that from a .scn.ron and yes that worked.

As for using the prefab system to load assets in, im not sure as bevy_asset_loader seems heavily coupled with bevy_common_assets for dynamic assets as they currently are. The other side of this could be to just use static assets with a specified path which overall may be a simpler short term solution.

Reply:
@rewin123 any ideas on this?